example-expo/AccessoryBar.js

```diff
@@ -11,11 +11,19 @@ import {
 export default class AccessoryBar extends React.Component {
   render() {
     const { onSend } = this.props
+    const isTyping = this.props.isTyping
```

Comment:
```jsx
const { onSend, isTyping } = this.props
```
?

Reply:
good catch. I pasted from my own code so I didn't have the onsend prop 😝